wolfcrypt/src/rsa.c

```diff
@@ -2095,17 +2114,265 @@ int wc_RsaFlattenPublicKey(RsaKey* key, byte* e, word32* eSz, byte* n,
     return 0;
 }
 
+
+static int RsaGetValue(mp_int* in, byte* out, word32* outSz)
```

Comment:
Should this function check "in" against NULL?  Seems like it could seg fault if key->e/n/d/p/q are NULL coming into this function.

Reply:
I'll add the check. The caller of this function is already checking the parameters before passing to `RsaGetValue()`.